app/controllers/tags_controller.rb

```diff
@@ -49,7 +49,8 @@ def update
 
   # DELETE /tags/1 or /tags/1.json
   def destroy
-    @tag.icon.purge
+    #@tag.icon.purge
+    @tag.posts.each {|post| post.destroy!}
```

Comment:
Should we keep it like this or should we remove the tag from each post? I think I like the latter

Reply:
I originally didn't remove them, but then the database had issue with the fact that the tag_id in a post led to a tag that didn't exist. I could change it so that all of the posts with a tag being destroyed have their tag changed to the 'null' tag of sorts.

Reply:
I added an update that should improve that functionality.